src/lib/form-field/_form-field-theme.scss

```diff
@@ -49,7 +49,7 @@
     }
   }
 
-  .mat-form-field-autofill-float:-webkit-autofill + .mat-form-field-placeholder,
```

Comment:
Can you instead just add the `mat-form-field-autofill-float` class to the `MdInput`? This was intended as a class users could add to custom a `MdFormFieldControl` to get the same effect, but I forgot to actually add it, haha...

Reply:
Wouldn't that involve injecting the parent `MdFormField` to see if the floating placeholder is enabled or not?

Reply:
Addressed discussed feedback from Slack.